internal/internal_worker_base.go

```diff
@@ -421,3 +422,20 @@ func (bw *baseWorker) Stop() {
 	}
 	return
 }
+
+func getAutoConfigHint(task interface{}) *shared.AutoConfigHint {
+	switch t := task.(type) {
```

Comment:
when I was poking through these types, to see if this was complete, I believe I found a fourth.  I don't remember if it was locally-dispatched activity types or query tasks or something else though.

what happens if this is wrong?  does it just consider [something] as having no cost?